.gitignore

```diff
@@ -8,5 +8,10 @@ dist-newstyle/
 *.prof
 *.hp
 
-# vendor directory
-vendor
\ No newline at end of file
+# Development files/directories
+vendor/
+sandbox/
```

Comment:
Why `sandbox`? Are multiple people using this convention? I'm personally already using `.scratch`, with my own folder structure for test projects.

Reply:
Just an idea.  I liked the idea of sandbox because of the pre-existing connotation of it being isolated.

Reply:
Given that we already have `make fossa`, I don't encoding the `sandbox` convention adds much value. I'd personally remove references to `sandbox`, but if you want to start this as a convention and drive any tooling around it, I'm okay with that too.

Reply:
We need somewhere that git will ignore, it's really annoying to have it always tell me there are untracked files.  I'll leave it, but I'll also make it more modular so we can decide on some other name if we want.  And this only affects us, so I'm not worried about committing to anything.

Reply:
If I develop other tooling it will be modular with that name as well.

Reply:
> We need somewhere that git will ignore

I don't think picking an arbitrary folder name is the right solution for this. Developers should configure their personal preferences. You can set up [personal repository exclusion settings](https://stackoverflow.com/questions/25912441/can-i-add-a-gitignore-file-just-for-me-that-overrides-the-project-gitignore), although I use a [personal global gitignore](https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file).

Reply:
I also avoid doing either of those exclusion methods, since it can be difficult to track down why something is excluded if it's not right there in the project.  Many people don't know about either of those features well enough to use them without googling it (though they are pretty simple once you look) and providing an obvious place to put test projects seems a lot simpler to me.  

Is picking an arbitrary name really so bad?  I don't see how this would have such a negative effect.  We could just remove it if no one else uses it.

Reply:
> Is picking an arbitrary name really so bad? I don't see how this would have such a negative effect. We could just remove it if no one else uses it.

I just think that this behavior ("hey there's a folder named `sandbox` that for some reason is ignored") is surprising, and that it provides little value given that developers can really easily set up personal gitignores. The articles I linked were the top 2 results for "personal gitignore".

I'm not going to fight hard to block this, but I don't think it's the right tool for this problem.